patches/chrome-utility-importer-profile_import_impl.cc.patch

```diff
@@ -19,7 +19,7 @@ index 1cbcc2fcb663c43daad9e6ad86f97e74b964aa11..2910707b7f569cb87947c81d8522f1d9
  #include "base/threading/thread.h"
  #include "base/threading/thread_task_runner_handle.h"
  #include "build/build_config.h"
-@@ -39,6 +43,16 @@ void ProfileImportImpl::StartImport(
+@@ -39,6 +43,19 @@ void ProfileImportImpl::StartImport(
```

Comment:
can we subclass `ProfileImportImpl` and only overrides `StartImport`?
In overridden method do OSCrypt password hack first and then call parent's StartImport
so we only need to substitute `ProfileImportImpl` to `BraveProfileImportImpl` by preprocessor in `chrome/utility/importer/profile_import_service.cc` with only one line

Reply:
There is one unrelated request but it will allow us to remove this patch completely
can we use preprocessor for `BraveExternalProcessImporterBridge` ?